src/libprojectM/Renderer/ShaderEngine.cpp

```diff
@@ -590,7 +590,22 @@ GLuint ShaderEngine::compilePresetShader(const PresentShaderType shaderType, Sha
 
             if (texDesc.first == NULL)
             {
-                std::cerr << "Texture loading error for: " << sampler << std::endl;
+                std::string extensionList = "";
+                // prepare a list of extensions we looked for, so it's more clear what a texture is in error logs.
```

Comment:
any extension will work, just the first part of the name should match. i suggest removing this code